modules/apps/27-interchain-accounts/host/keeper/genesis_test.go

```diff
@@ -4,15 +4,14 @@ import (
 	genesistypes "github.com/cosmos/ibc-go/v8/modules/apps/27-interchain-accounts/genesis/types"
 	"github.com/cosmos/ibc-go/v8/modules/apps/27-interchain-accounts/host/keeper"
 	"github.com/cosmos/ibc-go/v8/modules/apps/27-interchain-accounts/host/types"
+	icahosttypes "github.com/cosmos/ibc-go/v8/modules/apps/27-interchain-accounts/host/types"
 	icatypes "github.com/cosmos/ibc-go/v8/modules/apps/27-interchain-accounts/types"
+	host "github.com/cosmos/ibc-go/v8/modules/core/24-host"
 	ibctesting "github.com/cosmos/ibc-go/v8/testing"
 )
 
 func (suite *KeeperTestSuite) TestInitGenesis() {
```

Comment:
it's really hard to test both setups of capabilities already being initialized or not being initialized as the testing pkg will call init genesis first and the portID is hard coded. I personally think this sort of integration testing is better handled by the e2e's which is why I added coverage there